aquadoggo/src/graphql/utils.rs

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use async_graphql::dynamic::ResolverContext;
+use async_graphql::Value;
+use p2panda_rs::document::{DocumentId, DocumentViewId};
+use p2panda_rs::operation::OperationValue;
+use p2panda_rs::schema::SchemaId;
+use p2panda_rs::storage_provider::error::DocumentStorageError;
+use p2panda_rs::storage_provider::traits::DocumentStore;
+
+use crate::db::{types::StorageDocument, SqlStore};
+use crate::graphql::scalars::{DocumentIdScalar, DocumentViewIdScalar};
+
+const DOCUMENT_FIELDS_SUFFIX: &str = "Fields";
+
+// Correctly formats the name of a document field type.
+pub fn fields_name(schema_id: &SchemaId) -> String {
+    format!("{}{DOCUMENT_FIELDS_SUFFIX}", schema_id)
+}
+
+/// Convert non-relation operation values into GraphQL values.
+///
+/// Panics when given a relation field value.
+pub fn gql_scalar(operation_value: &OperationValue) -> Value {
+    match operation_value {
+        OperationValue::Boolean(value) => value.to_owned().into(),
+        OperationValue::Integer(value) => value.to_owned().into(),
+        OperationValue::Float(value) => value.to_owned().into(),
+        OperationValue::String(value) => value.to_owned().into(),
+        // Recursion not implemented yet.
+        _ => todo!(),
```

Comment:
What is this about? 🤔

Reply:
Ah yeah originally I thought I'd implement the relation logic here, but actually no... This can either just be a panic, or move the method content in place, I think it's only used in one place.

Reply:
This method is only used for converting concrete values, not relations.